### PR TITLE
Feature/automatic login if user is already logged in

### PR DESCRIPTION
**Pull request by SEEMADHAMI**

### CODE REVIEW

File: lib/screens/login_screen.dart [hunk 1]
```diff
@@ -1,211 +1,129 @@
...
-  bool _isProcessing = false;
-
-  Future<FirebaseApp> _initializeFirebase() async {
-    FirebaseApp firebaseApp = await Firebase.initializeApp();
+  final _formKey = GlobalKey<FormState>();
+  final emailController = TextEditingController();
+  final passwordController = TextEditingController();
 
-    User? user = FirebaseAuth.instance.currentUser;
+  final _auth = FirebaseAuth.instance;
```
Comment (Hpaneru) [inline lib/screens/login_screen.dart]: remove this initialization.

**Review by Hpaneru — CHANGES_REQUESTED** [review]
resolve the comments

File: lib/screens/signup_screen.dart [hunk 2]
```diff
@@ -1,149 +1,118 @@
...
 class _SignUpScreenState extends State<SignUpScreen> {
-  final _registerFormKey = GlobalKey<FormState>();
+  bool loading = false;
+  final _formKey = GlobalKey<FormState>();
+  final emailController = TextEditingController();
+  final passwordController = TextEditingController();
 
-  final _emailTextController = TextEditingController();
-  final _passwordTextController = TextEditingController();
+  FirebaseAuth _auth = FirebaseAuth.instance;
```
Comment (Hpaneru) [inline lib/screens/signup_screen.dart]: remove this from here as well

Comment (SEEMADHAMI) [reply lib/screens/signup_screen.dart]: Removed

**Review by Hpaneru — APPROVED** [review]
LGTM

**Pull request merged**